openpype/modules/default_modules/ftrack/event_handlers_user/action_create_project_structure.py

```diff
@@ -2,8 +2,8 @@
 import re
 import json
 
-from openpype_modules.ftrack.lib import BaseAction, statics_icon
-from openpype.api import Anatomy, get_project_settings
+from openpype.modules.ftrack.lib import BaseAction, statics_icon
```

Comment:
Modules import has changed. Use:
```suggestion
from openpype_modules.ftrack.lib import BaseAction, statics_icon
```

Reply:
done